### ISSUE TITLE
Multiple Instances

### ISSUE THREAD

**Issue by blakeembrey** [blakeembrey/pluralize#25]
It'd be cool to create multiple `pluralize` instances. The use-case that's been brought up before is acronyms (so perhaps have an option here for output case or a function that handles sanitization manually?). However, I think multiple locales would be perfect too. E.g. `require('pluralize/de')`. If anyone is willing to try create automated pluralizations for other locales that'd would be very cool.

**Comment by dominykas** [comment]
Hi,

any update on this? I'm happy to PR whatever is a preferred approach.

My personal preference would be to either have some sort of a `preserveCase` option - I think there are valid scenarios where people would want to simply turn off this behavior, because anything that's uppercase is not really a dictionary word (e.g. on the web my preferred method of capitalizing stuff would be `text-transform: uppercase`, rather than have capitalized text strings somewhere - I'd probably never need it).

**Comment by FranzSkuffka** [comment]
+1

**Comment by blakeembrey** [comment]
I think it could be once it’s turned into a flag, as you can then you use the function for this. I’ll be sure to keep it in mind when I work on this, likely this upcoming weekend.

**Comment by jensbodal** [comment]
Yeah the real issue is everything gets passed into the restoreCase function. I feel that the ability to write case-sensitive rules should be allowed.
